### ISSUE TITLE
show-hide-applets@mohammad-sn: does not work with multi-monitor setup

### ISSUE THREAD

**Issue by sphh** [linuxmint/cinnamon-spices-applets#1506]
```
 * Applet version/Build date: Last modified 2017-10-02
 * Cinnamon version: 3.6.6
 * Distribution: Mint 18.3
 * Graphics hardware *and* driver used
 * Snippet or attachment of ~/.cinnamon/glass.log (grep "show-hide" .cinnamon/glass.log): info t=2017-11-28T09:47:28.125Z Loaded applet show-hide-applets@mohammad-sn in 7 ms
 * Snippet or attachment of ~/.xsession-errors (grep "show-hide" .xsession-errors"): <empty>

 ```

**Notify author of applet**
@mohammad-sn 

**Issue**
After today's upgrade to Linux Mint 18.3, the `show-hide-applet` does not hide the applets positioned left to it anymore.

**Steps to reproduce**
1. Upgrade to Linux Mint 18.3.
2. Install `show-hide-applet`.
3. Activate Panel edit mode.
4. Position the `cinnamon-timer@jake1164`, `sound@cinnamon.org`, `removable-drives@cinnamon.org` and `power@cinnamon.org` applets left to the `show-hide-applet`.

**Expected behaviour**
The four applets from step 4 above should hide/unhide whenever the triangle is clicked. The triangle changes shape from left to right, but does not hide the applets.

**Comment by NikoKrause** [comment]
Restart your machine. Try it again. If it still doesn't work, post a screenshot of the applets `About...` dialog.

**Comment by sphh** [comment]
Here is the screenshot after I restarted the box: ![image](https://user-images.githubusercontent.com/4856462/33320004-a75f47b6-d438-11e7-8810-b65356b645a4.png)

Here after I clicked on the icon: ![image](https://user-images.githubusercontent.com/4856462/33320087-eea71c98-d438-11e7-8774-e95247d90576.png)

And here in panel edit mode: ![image](https://user-images.githubusercontent.com/4856462/33320116-0c4d6842-d439-11e7-9cfe-953b76deb562.png)

**Comment by NikoKrause** [comment]
It works fine here.
You can try to uninstall it and then reinstall.

**Comment by sphh** [comment]
I already removed it entirely and reinstalled it. I did it again and it does not work.

But I found the culprit: I have a dual monitor setup with one panel across each monitor. If I put it on one panel it _does not_ work, if I put it on the other panel it _does_ work. If I move the panel where this `show-hide-applets` applet works to the other monitor (now having one panel at the top and one at the bottom), it still works on this panel, but not on the other and vice versa.

If the cause cannot be found, a (not so) quick workaround might be, that I swap my two panels, delete all applets and adding them again to the other panel ...
